### PR TITLE
[risk=no][RW-8917] Remove "request an extension of your initial credits" from initial credits emails

### PR DESCRIPTION
**Pull request by jmthibault79**
* rename "free tier" to "initial credits" in MailService
* reorganize emails

Tested locally.

Passed threshold email:

---

email-thresh

---

Expiration email:

---

email-expired

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java [hunk 1]
```diff
@@ -329,7 +329,7 @@ private void maybeAlertOnCostThresholds(
 
     if (remainingBalance < 0 && previousRemainingBalance > 0) {
       try {
-        mailService.alertUserFreeTierExpiration(user);
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java:332]: TODO (too big for this PR): rename all "free tier" instances in FreeTierBillingService, plus the service itself

File: api/src/main/java/org/pmiops/workbench/mail/EmailSubstitutionField.java [hunk 2]
```diff
@@ -2,45 +2,45 @@
 
 // used by MailServiceImpl to make template substitutions
 public enum EmailSubstitutionField {
+  ACTION("ACTION"),
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/mail/EmailSubstitutionField.java:5]: alphabetized

File: api/src/main/java/org/pmiops/workbench/mail/EmailSubstitutionField.java [hunk 3]
```diff
@@ -2,45 +2,45 @@
...
+  DISK_UNUSED_DAYS("DISK_UNUSED_DAYS"),
+  EGRESS_REMEDIATION_DESCRIPTION("EGRESS_REMEDIATION_DESCRIPTION"),
   EXPIRATION_DATE("EXPIRATION_DATE"),
-  REGISTRATION_IMG("REGISTRATION_IMG"),
-  INSTRUCTIONS("INSTRUCTIONS"),
-  FREE_CREDITS_RESOLUTION("FREE_CREDITS_RESOLUTION"),
+  FIRST_NAME("FIRST_NAME"),
+  FROM_EMAIL("FROM_EMAIL"),
+  HEADER_IMG("HEADER_IMG"),
+  INITIAL_CREDITS_RESOLUTION("INITIAL_CREDITS_RESOLUTION"),
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/mail/EmailSubstitutionField.java:23]: was FREE_CREDITS_RESOLUTION

File: api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java [hunk 4]
```diff
@@ -57,25 +57,25 @@ public class MailServiceImpl implements MailService {
 
   private static final Logger log = Logger.getLogger(MailServiceImpl.class.getName());
 
-  private static final String RAB_SUPPORT_EMAIL = "aouresourceaccess@od.nih.gov";
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java:60]: moved below email resources

File: api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java [hunk 5]
```diff
@@ -57,25 +57,25 @@ public class MailServiceImpl implements MailService {
...
-
-  private static final String WELCOME_RESOURCE = "emails/welcomeemail/content.html";
-  private static final String INSTRUCTIONS_RESOURCE = "emails/instructionsemail/content.html";
-  private static final String FREE_TIER_DOLLAR_THRESHOLD_RESOURCE =
-      "emails/dollarthresholdemail/content.html";
-  private static final String FREE_TIER_EXPIRATION_RESOURCE =
-      "emails/freecreditsexpirationemail/content.html";
-  private static final String REGISTERED_TIER_ACCESS_THRESHOLD_RESOURCE =
-      "emails/rt_access_threshold_email/content.html";
+  private static final String EGRESS_REMEDIATION_RESOURCE =
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java:60]: alphabetized

File: api/src/main/resources/emails/initial_credits_dollar_threshold/content.html [hunk 6]
```diff
@@ -31,7 +31,7 @@
 <div>Hello ${FIRST_NAME} ${LAST_NAME},</div>
 <div>This is a reminder that your account ${USERNAME} has used ${USED_CREDITS} in initial credits offered by the All of Us Program.
   The remaining balance on your initial credits is ${CREDIT_BALANCE}.</div>
-<div>Once you use all of your initial credits, ${FREE_CREDITS_RESOLUTION}</div>
-<div>If you have any questions about this email or want to request an extension of your initial credits, please contact support@researchallofus.org.</div>
+<div>Once you use all of your initial credits, you will not be able to use the analysis tools in the Workbench without a valid billing account. To continue your analyses, ${INITIAL_CREDITS_RESOLUTION}</div>
```
Comment (jmthibault79) [inline api/src/main/resources/emails/initial_credits_dollar_threshold/content.html:34]: reworded to match expiration

File: api/src/main/resources/emails/initial_credits_dollar_threshold/content.html [hunk 7]
```diff
@@ -31,7 +31,7 @@
 <div>Hello ${FIRST_NAME} ${LAST_NAME},</div>
 <div>This is a reminder that your account ${USERNAME} has used ${USED_CREDITS} in initial credits offered by the All of Us Program.
   The remaining balance on your initial credits is ${CREDIT_BALANCE}.</div>
-<div>Once you use all of your initial credits, ${FREE_CREDITS_RESOLUTION}</div>
-<div>If you have any questions about this email or want to request an extension of your initial credits, please contact support@researchallofus.org.</div>
```
Comment (jmthibault79) [inline api/src/main/resources/emails/initial_credits_dollar_threshold/content.html:35]: cannot request extension

File: api/src/main/resources/emails/initial_credits_expiration/content.html [hunk 8]
```diff
@@ -30,7 +30,7 @@
 <img src="${HEADER_IMG}">
 <div>Hello ${FIRST_NAME} ${LAST_NAME},</div>
 <div>This is a notification that the initial credits offered to your account ${USERNAME} by the All of Us Program have expired.</div>
-<div>You will not be able to use the analysis tools in the Workbench without a valid billing account. To continue your analyses, ${FREE_CREDITS_RESOLUTION}</div>
-<div>If you have any questions about this email or want to request an extension of your initial credits, please contact support@researchallofus.org.</div>
```
Comment (jmthibault79) [inline api/src/main/resources/emails/initial_credits_expiration/content.html:34]: cannot request extension

File: api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java [hunk 9]
```diff
@@ -429,13 +431,12 @@ private Map<EmailSubstitutionField, String> instructionsSubstitutionMap(
         .build();
   }
 
-  private String getFreeCreditsResolutionText() {
-    return "you can request additional free credits by contacting support "
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java:433]: cannot request additional credits

File: api/src/main/resources/emails/initial_credits_dollar_threshold/content.html [hunk 10]
```diff
@@ -31,7 +31,7 @@
...
+<div>Once you use all of your initial credits, you will not be able to use the analysis tools in the Workbench without a valid billing account. ${INITIAL_CREDITS_RESOLUTION}</div>
```
Comment (jmthibault79) [inline api/src/main/resources/emails/initial_credits_dollar_threshold/content.html:34]: copied expiration text

File: api/src/test/java/org/pmiops/workbench/billing/FreeTierBillingServiceTest.java [hunk 11]
```diff
@@ -146,7 +146,7 @@ public void checkFreeTierBillingUsage_exceedsDollarThresholds() throws Messaging
     doReturn(mockBQTableSingleResult(costOverThreshold)).when(bigQueryService).executeQuery(any());
     freeTierBillingService.checkFreeTierBillingUsageForUsers(Sets.newHashSet(user));
     verify(mailService)
-        .alertUserFreeTierDollarThreshold(
```
Comment (NehaBroad) [inline api/src/test/java/org/pmiops/workbench/billing/FreeTierBillingServiceTest.java:149]: Rename the class?

**Review by NehaBroad — APPROVED** [review]

**Pull request merged**